Lab Rat - GWJ 2020/project.godot

```diff
@@ -17,11 +17,18 @@ _global_script_class_icons={
 
 config/name="Lab Rat - GWJ 2020"
 run/main_scene="res://Scenes/splash_screen/splash_screen.tscn"
-boot_splash/fullsize=false
 boot_splash/use_filter=false
 boot_splash/bg_color=Color( 0, 0, 0, 1 )
 config/icon="res://icon.png"
 
+[autoload]
+
+conami="*res://Scenes/conami_autoload.gd"
```

Comment:
Keine updates mehr gezogen? Das ist schon seit einiger Zeit nicht mehr im Autoload.

Reply:
Ist aber nicht schlimm, das conami_autoload.gd script ist jetzt direkt in der szene drin, kannst also auch super darauf zugreifen.

Reply:
Habe es wieder reingemacht, damit man es auch direkt abrufen kann.

Reply:
Über welche Node?